internal/config/links.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"time"
+
+	"gitlab.com/distributed_lab/figure"
+	"gitlab.com/distributed_lab/kit/kv"
+	"gitlab.com/distributed_lab/logan/v3/errors"
+)
+
+type Links struct {
+	Duration time.Duration `fig:"duration"`
+}
+
+func (c *config) Links() Links {
+	c.Lock()
+	defer c.Unlock()
```

Comment:
use `comfig.Once` for configuration. It already has mutex and logic for singleton configuration.